PABC.MigrationService/Program.cs

```diff
@@ -14,7 +14,16 @@
 var builder = Host.CreateApplicationBuilder(args);
 
 builder.AddServiceDefaults();
-builder.AddNpgsqlDbContext<PabcDbContext>(connectionName: "Pabc");
+builder.AddNpgsqlDbContext<PabcDbContext>(
+    connectionName: "Pabc",
+    configureDbContextOptions: options =>
+    {
+        options.UseNpgsql(npgsqlOptions =>
+        { 
+            npgsqlOptions.MigrationsHistoryTable("__ef_migrations_history", "public");
```

Comment:
I think specifically the MigrationsHistoryTable needs to keep the same name, otherwise ef core tries to execute migrations from before this change again. So we can remove this line